activerecord/lib/active_record/errors.rb

```diff
@@ -178,10 +178,9 @@ class Rollback < ActiveRecordError
   class DangerousAttributeError < ActiveRecordError
   end
 
-  UnknownAttributeError = ActiveSupport::Deprecation::DeprecatedConstantProxy.new( # :nodoc:
-    'ActiveRecord::UnknownAttributeError', 
-    'ActiveModel::AttributeAssignment::UnknownAttributeError'
-  )
+  # Raised when unknown attributes are supplied via mass assignment.
+  class UnknownAttributeError < ActiveModel::UnknownAttributeError
```

Comment:
@bogdan : We decided to finally alias the constant as there was no big deal ; furthermore, we are already raising errors with constants nested under ActiveModel so having it in the output of `error.class.to_s` is not that unexpected.